src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/InMemoryEmbeddingServiceConfigStore.java

```diff
@@ -0,0 +1,27 @@
+package io.stargate.sgv2.jsonapi.service.embedding.configuration;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import io.quarkus.arc.lookup.LookupIfProperty;
+import jakarta.enterprise.context.ApplicationScoped;
+import java.util.Optional;
+
+@LookupIfProperty(name = "stargate.jsonapi.embedding.config.store", stringValue = "in-memory")
+@ApplicationScoped
+public class InMemoryEmbeddingServiceConfigStore implements EmbeddingServiceConfigStore {
```

Comment:
@tatu-at-datastax : I needed something like this to finish the `createEmbeddingService`. This will eventually this will change to some security manager store. That will be separate PR once we have details on how it's going to be hasted.